### PR TITLE
erase unnecessary lines

### PR DESCRIPTION
**Pull request by sora-ryu**
We can remove some unnecessary lines to make more neat.

### CODE REVIEW

**Comment by elizabethjmohr** [comment]
Amazing job!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,2 @@
 # URSSI-Workshop
 This is for practicing pull request and code review
```
Comment (elizabethjmohr) [inline README.md:2]:
```suggestion
This is for practicing pull request and code review
More text blah blah blah
```

**Review by elizabethjmohr — COMMENTED** [review]
Overall great job! I just added one suggestion.

File: README.md (hunk 1 above)
Comment (sora-ryu) [reply README.md:2]: Thank you so much! That looks awesome!!

**Pull request merged**